sdk/openai/azure-ai-openai/src/test/java/com/azure/ai/openai/responses/AzureResponsesTestBase.java

```diff
@@ -40,8 +46,10 @@
 
 public class AzureResponsesTestBase extends TestProxyTestBase {
 
+    private static final String MS_LOGO_PNG = "ms_logo.png";
+
     private ResponsesClientBuilder getBuilderForTests(HttpClient httpClient,
-        AzureResponsesServiceVersion serviceVersion) {
+                                                      AzureResponsesServiceVersion serviceVersion) {
```

Comment:
I think the style checker will complain about this text formatting. If you run `mvn clean install` it will correct this indentation for you.